packages/webview_flutter/webview_flutter_platform_interface/lib/src/types/http_response_error.dart

```diff
@@ -0,0 +1,45 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/foundation.dart';
+
+/// Error returned in `PlatformNavigationDelegate.setOnHttpError` when an HTTP
+/// response error has been received.
+/// 
```

Comment:
Format test is failing due to this extra space:

```suggestion
///
```